metricflow/test/model/validations/test_configurable_rules.py

```diff
@@ -21,11 +22,18 @@ def test_can_configure_model_validator_rules(simple_model__pre_transforms: UserC
     issues = ModelValidator().validate_model(model).issues
     assert len(issues) == 1, f"ModelValidator with default rules had unexpected number of issues {issues}"
 
-    # confirm that with an empty configuration, no issue is raised
-    issues = ModelValidator(rules=[]).validate_model(model).issues
-    assert len(issues) == 0, f"Empty rules list configured ModelValidator returned issues {issues}"
-
     # confirm that a custom configuration excluding DataSourceMeasuresUniqueRule, no issue is raised
     rules = [rule for rule in ModelValidator.DEFAULT_RULES if rule.__class__ is not ValidMaterializationRule]
     issues = ModelValidator(rules=rules).validate_model(model).issues
     assert len(issues) == 0, f"ModelValidator without ValidMaterializationRule returned issues {issues}"
+
+
+def test_cant_configure_model_validator_without_rules() -> None:  # noqa: D
+    with pytest.raises(ValueError):
+        ModelValidator(rules=[])
+
+    with pytest.raises(ValueError):
+        ModelValidator(rules=())
+
+    with pytest.raises(ValueError):
+        ModelValidator(rules=None)  # type: ignore
```

Comment:
I think you can remove this - this case is not supported according to the API so there's no need for the added test.

Reply:
Yea I mostly put it in because I expect someone to do something like `ModelValidator(rules=None)  # type: ignore` or similar as a work around something, I know I've done things like this myself in the past. In the case that someone does, it should and will throw the `ValueError` exception.